doc/conf.py

```diff
@@ -343,28 +330,28 @@
     ('boards/arm/msp_exp432p401r_launchxl/doc/msp_exp432p401r_launchxl', 'boards/arm/msp_exp432p401r_launchxl/doc/index'),
     ('boards/arm/nrf51_ble400/doc/nrf51_ble400', 'boards/arm/nrf51_ble400/doc/index'),
     ('boards/arm/nrf51_blenano/doc/nrf51_blenano', 'boards/arm/nrf51_blenano/doc/index'),
-    ('boards/arm/nrf51_pca10028/doc/nrf51_pca10028', 'boards/arm/nrf51dk_nrf51422/doc/index'),
     ('boards/arm/nrf51_pca10028/doc/index', 'boards/arm/nrf51dk_nrf51422/doc/index'),
+    ('boards/arm/nrf51_pca10028/doc/nrf51_pca10028', 'boards/arm/nrf51dk_nrf51422/doc/index'),
```

Comment:
Just curious.
The pca board were deprecated before 2.3, and completely removed with this PR: https://github.com/zephyrproject-rtos/zephyr/pull/28763

How long do we want to keep the redirection link for the old boards ?

Reply:
> How long do we want to keep the redirection link for the old boards ?

It's a fair question, but it's out of scope for this PR.